.werft/observability/monitoring-satellite.ts

```diff
@@ -56,7 +58,9 @@ export async function installMonitoringSatellite(params: InstallMonitoringSatell
     ensureIngressesReadiness(params)
 }
 
-async function ensureCorrectInstallationOrder(){
+async function ensureCorrectInstallationOrder() {
+    const werft = getGlobalWerftInstance()
```

Comment:
Instead of doing a get on every function, why not create a "global" one at the top of this file and reuse it everywhere?

Reply:
By hiding the instance behind a method I can check if the instance is properly instantiated and throw an error if it isn't ☺️ I agree that it is a little bit awkward having to get the instance like this, but I think the proper way to fix it is to slowly move our Typescript code towards more idiomatic typescript where modules that have dependencies are represented as classes and then rely on "dependency injection" techniques to glue things together so for now I'm sticking with using a method like this unless you strongly disagree ☺️

Reply:
No strong opinions :) 

Lets improve it on next interations